docs/howto/account-billing/forgot-your-password.md

```diff
@@ -0,0 +1,55 @@
+# Resetting your password or reclaiming your username
+
+Via the {{gui}}, you can reset a lost password or reclaim your username.
+
+## Reset lost password
+
+Begin by navigating to the https://{{gui_domain}}. At the bottom left-hand side of the page, click on the Forgot your password? button.
+
+![Picture with a orange button with white text Forgot your password? framed with red color](assets/Forgot-your-password.png)
+
+A pop-up window titled Reclaim lost password!, appears. Type in your username, then click on the Reclaim password! button.
```

Comment:
Please put double quotes around "Reclaim lost password!" and "Reclaim password!".